akd/src/errors.rs

```diff
@@ -13,33 +13,33 @@ use crate::node_state::NodeLabel;
 /// Symbolizes a AkdError, thrown by the akd.
 #[derive(Debug)]
 pub enum AkdError {
-    /// Error propogation
-    HistoryTreeNodeErr(HistoryTreeNodeError),
-    /// Error propogation
-    DirectoryErr(DirectoryError),
-    /// Error propogation
+    /// Error propagation
```

Comment:
I feel like the comments, which are exported in the crate publicly could be cleaned up too over "error propogation"

Reply:
Are they not auto-synced? I can look into it if not.

Reply:
I believe they're hand typed... no? Maybe there's a rust feature I'm unaware of? Check with intellisense maybe and see what it says for a description on that case.

Reply:
`cargo doc` generates the docs and puts them into `target/doc `. We can specify a website that serves these docs. (See `[documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-documentation-field)`). 

Alternatively, if we don't, `docs.rs` automatically builds the docs and serves them. So doc updates in the code should be reflected in the published repo.